vignettes/data_analysis_single_dose_treatment_workflow.Rmd

```diff
@@ -257,15 +258,15 @@ diff_abudance_data <- data_filtered_uniprot %>%
 The p-value calculated with the t-test is automatically adjusted for multiple testing using the Benjamini-Hochberg (FDR) correction. This assures that we keep the false discovery rate low. An assumption of this correction is however, that p-values should have an overall even distribution with a concentration of values in the significant range. You can check this by using the protti function `plot_pval_distribution()`.
 
 ```{r pval_distribution, message = FALSE, warning = FALSE, fig.align= "center", fig.width = 6, fig.height = 5}
-plot_pval_distribution(data = diff_abudance_data,
+plot_pval_distribution(data = diff_abundance_data,
                        grouping = eg_precursor_id,
                        pval = pval
                        )
 ```
 
 For this subset of data the distribution of p-values is relatively equal and there is no increase in values in the low p-value range but rather a decrease. This means that the Benjamini-Hochberg algorithm might not be suitable for correcting these p-values. You can pick another algorithm for the correction and supply it to the `p_adj_method` argument and run `diff_abundance()` again. The available methods are any methods that can be supplied to the standard R function `p.adjust()` that is used internally. You can also just use this function or any other correction algorithm on your data and overwrite the values in the `adj_pval` column.
 
-_Note: If the p-values are not fulfilling the assumption of the multiple-testing correction, meaning very little observations are changing significantly, it can happen that only a few significant hits remain after adjusting the p-value. This can be a common occurrence so you should always have a look at the p-value distribution after statistical testing._
+_Note: If the p-values are not fulfilling the assumption of the multiple-testing correction, meaning very little observations are changing significantly, it can happen that only a few significant hits remain after adjusting the p-value. This can be a common occurrence so you should always have a look at the p-value distribution after statistical testing and before adjusting your p-values. This will help you decide on whether or not you should adjust your p-values._
```

Comment:
The p-values are always automatically adjusted with the function. It is just a question wether or not you should trust the adjustment. In this case you cannot test before adjusting the p-value because it is done in the same step. So maybe it is a bit confusing to say "before adjusting your p-values". You should basically check and see if you need to adjust it again differently.